af/fastpath/debian/postinst

```diff
@@ -9,13 +9,6 @@ case "$1" in
         adduser --system --quiet --ingroup fastpath --no-create-home --home /var/lib/fastpath fastpath
         mkdir -p /var/lib/fastpath/ssh/
         chown fastpath /var/lib/fastpath
-        if [ ! -f /var/lib/fastpath/ssh/id_ed25519 ]
-        then
-          logger "fastpath - Generating SSH key"
-          # Create key suitable for paramiko. The trivial password protecs the privkey from being used on
-          # the wrong host or somehow exposed by mistake
-          ssh-keygen -t ed25519 -f /var/lib/fastpath/ssh/id_ed25519 -N "$(cat /etc/machine-id)"
-        fi
```

Comment:
I assume this is a cleanup action now that the ssh fetcher is gone

Reply:
yep